engine/apps/slack/scenarios/distribute_alerts.py

```diff
@@ -141,22 +141,6 @@ def _post_alert_group_to_slack(
                 channel_id=channel_id,
             )
 
-            # If alert was made out of a message:
-            if alert_group.channel.integration == AlertReceiveChannel.INTEGRATION_SLACK_CHANNEL:
-                channel = json.loads(alert.integration_unique_data)["channel"]
-                result = self._slack_client.chat_postMessage(
-                    channel=channel,
-                    thread_ts=json.loads(alert.integration_unique_data)["ts"],
-                    text=":rocket: <{}|Incident registered!>".format(alert_group.slack_message.permalink),
-                    team=slack_team_identity,
-                )
-                alert_group.slack_messages.create(
-                    slack_id=result["ts"],
-                    organization=alert_group.channel.organization,
-                    _slack_team_identity=self.slack_team_identity,
-                    channel_id=channel,
-                )
```

Comment:
while working on https://github.com/grafana/oncall-private/issues/2947, I was digging around here and I'm _fairly_ certain `INTEGRATION_SLACK_CHANNEL` is legacy/deprecated/can-be-removed?

Reply:
Yes I _think_ it was removed in https://github.com/grafana/oncall/pull/2383